sql/core/src/test/resources/sql-tests/inputs/postgreSQL/window_part3.sql

```diff
@@ -12,17 +12,16 @@ CREATE TABLE empsalary (
     enroll_date date
 ) USING parquet;
 
-INSERT INTO empsalary VALUES
-('develop', 10, 5200, '2007-08-01'),
-('sales', 1, 5000, '2006-10-01'),
-('personnel', 5, 3500, '2007-12-10'),
-('sales', 4, 4800, '2007-08-08'),
-('personnel', 2, 3900, '2006-12-23'),
-('develop', 7, 4200, '2008-01-01'),
-('develop', 9, 4500, '2008-01-01'),
-('sales', 3, 4800, '2007-08-01'),
-('develop', 8, 6000, '2006-10-01'),
-('develop', 11, 5200, '2007-08-15');
+INSERT INTO empsalary VALUES ('develop', 10, 5200, cast ('2007-08-01' as date));
+INSERT INTO empsalary VALUES ('sales', 1, 5000, cast ('2006-10-01' as date));
+INSERT INTO empsalary VALUES ('personnel', 5, 3500, cast ('2007-12-10' as date));
+INSERT INTO empsalary VALUES ('sales', 4, 4800, cast ('2007-08-08' as date));
+INSERT INTO empsalary VALUES ('personnel', 2, 3900, cast ('2006-12-23' as date));
+INSERT INTO empsalary VALUES ('develop', 7, 4200, cast ('2008-01-01' as date));
+INSERT INTO empsalary VALUES ('develop', 9, 4500, cast ('2008-01-01' as date));
+INSERT INTO empsalary VALUES ('sales', 3, 4800, cast ('2007-08-01' as date));
+INSERT INTO empsalary VALUES ('develop', 8, 6000, cast ('2006-10-01' as date));
+INSERT INTO empsalary VALUES ('develop', 11, 5200, cast ('2007-08-15' as date));
```

Comment:
Instead of the above, the following?
```
INSERT INTO empsalary VALUES
 ('develop', 10, 5200, date '2007-08-01'),
 ('sales', 1, 5000, date '2006-10-01'),
 ('personnel', 5, 3500, date '2007-12-10'),
 ('sales', 4, 4800, date '2007-08-08'),
 ('personnel', 2, 3900, date '2006-12-23'),
 ('develop', 7, 4200, date '2008-01-01'),
 ('develop', 9, 4500, date '2008-01-01'),
 ('sales', 3, 4800, date '2007-08-01'),
 ('develop', 8, 6000, date '2006-10-01'),
 ('develop', 11, 5200, date '2007-08-15');
```